.gitignore

```diff
@@ -1,15 +1,2 @@
-# Binaries for programs and plugins
-*.exe
-*.exe~
-*.dll
-*.so
-*.dylib
-
-# Test binary, built with `go test -c`
-*.test
-
-# Output of the go coverage tool, specifically when used with LiteIDE
-*.out
-
-# Dependency directories (remove the comment below to include it)
-# vendor/
+.idea
```

Comment:
Let's comment this in similar way it's been done before and add a slash to signify that `.idea` is a directory:

```suggestion
# IntelliJ IDEA
.idea/
```